archetypes/Pools/FilterSelects/Bar.tsx

```diff
@@ -41,126 +41,128 @@ const REBALANCE_OPTIONS_MOBILE = [
     },
 ];
 
-const DENOTION_OPTIONS = [
+const DENOTATION_OPTIONS = [
     {
         key: DeltaEnum.Percentile,
-        text: 'Percent',
+        text: (
+            <>
+                <div className="mr-2">Relative</div>
+                <ArrowDown className="text-red-600 w-6" />
+                <ArrowDown className="rotate-180 text-green-600 w-6" />
+            </>
+        ),
     },
     {
         key: DeltaEnum.Numeric,
-        text: 'Number',
+        text: (
+            <>
+                <div className="mr-2">Absolute</div>
+                <ArrowDown className="text-red-600 w-6" />
+                <ArrowDown className="rotate-180 text-green-600 w-6" />
+            </>
+        ),
     },
 ];
 
 const FilterSelects: React.FC<FilterSelectsProps> = ({ state, dispatch }) => {
     return (
         <section className="container px-4 sm:px-0">
-            {/** Desktop */}
-            <div className="block lg:flex w-full mb-2">
-                <div className="xl:hidden">
-                    <TWButtonGroup
-                        value={state.rebalanceFocus}
-                        size="lg"
-                        onClick={(option) => dispatch({ type: 'setRebalanceFocus', focus: option as RebalanceEnum })}
-                        color={'tracer'}
-                        options={REBALANCE_OPTIONS_MOBILE}
-                    />
-                </div>
-                <div className="hidden xl:block">
-                    <TWButtonGroup
-                        value={state.rebalanceFocus}
-                        size="lg"
-                        onClick={(option) => dispatch({ type: 'setRebalanceFocus', focus: option as RebalanceEnum })}
-                        color={'tracer'}
-                        options={REBALANCE_OPTIONS_DESKTOP}
-                    />
+            <div className="w-full mb-2">
+                <div className="flex">
+                    <div className="hidden lg:flex mr-4 flex-col">
+                        <h3 className="mb-1 text-theme-text">Market</h3>
+                        <Dropdown
+                            variant="blue"
```

Comment:
Is this supposed to be blue?
https://www.figma.com/file/Gh6WzO8gLZWDmTvRbw8ikm/Tracer-Style-Guide-%26-Components?node-id=5993%3A214837

Reply:
yep, this link is outdated